.circleci/config.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           key: v1-deps-server-cache-{{ checksum "./server/yarn.lock" }}
       - run:
           name: Test server
-          command: yarn test:server
+          command: cd server && yarn && yarn test
```

Comment:
why are you doing the cd server when you have already set the working_directory?

Reply:
> why are you doing the cd server when you have already set the working_directory?

Similar issue with test server job. Attaching build failed error
![Uploading Screenshot from 2019-09-19 21-35-45.png…]()